### ISSUE TITLE
Blank window in all Jetbrains applications.

### ISSUE THREAD

**Issue by rahatzamancse** [baskerville/bspwm#1044]
I have tried running Android Studio, Pycharm and CLion. But after starting, they all end up in a grey window. I can tile them, float them, even resize them. They are not stuck because I get feedback from the window, and can close them (That is why I think, the application works, but just not rendering in the display properly). Example screenshot is here:
![Pycharm-problem](https://user-images.githubusercontent.com/22135181/71330099-100da900-2555-11ea-8e63-7a9a72d239de.png)
Running Manjaro-bspwm. All my config files can be found [here](https://github.com/rahatzamancse/my-linux-dots-git).
Please ask if you need any more information.

**Comment by mighty9245** [comment]
try this
https://wiki.archlinux.org/index.php/Java#Gray_window,_applications_not_resizing_with_WM,_menus_immediately_closing

**Comment by jallbrit** [comment]
@rahatzamancse did the solution(s) from the ArchWiki work for you?

**Comment by rahatzamancse** [comment]
Yes. 

I firstly used `wmname LG3D`. But this `wmname` trick brought more problems than it fixed. After using `wmname LG3D`, most of the dock like application did not work properly. For example [plank](https://launchpad.net/plank) was not able to focus a window on another workspace after clicking on the icon. Same thing happened also for xfce-panel and Latte dock (I tried those myself). Only [cairo-dock](http://glx-dock.org/) was able to focus on a window on another desktop.

So I used cairo-dock for a month (Without realizing that the reason was putting `wmname LG3D` in one of my init files. Once I tried the other solutions, `export AWT_TOOLKIT=MToolkit` and `export _JAVA_AWT_WM_NONREPARENTING=1`, they worked perfectly.

Sorry, I actually forgot to close the issue. Closing now.
